services/app/src/pages/game/Game.jsx

```diff
@@ -20,21 +22,28 @@ import {
 import Header from '../../components/header/Header';
 import '../../static/style/game.css';
 
-const Room = () => {
-  const [roomUrl] = useState(window.location.href);
+function Room() {
```

Comment:
Either rename Room to Game or Game to Room to have matching names between component and file